### ISSUE TITLE
Accessibility Insights icon on the banner of event configuration pop up has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#864]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode
- High Contrast ```#1```

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![11 1](https://user-images.githubusercontent.com/22068434/88347443-8ba82200-ccff-11ea-8477-910435aef215.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
I've asked design for an updated icon that incorporates a contrasting border.

**Comment by DaveTryon** [comment]
@iamrafan, this has been fixed in the release available at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1412.01. Please validate it at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1429.01.
